src/components/extractionQc/ExtractionQcForm.vue

```diff
@@ -0,0 +1,74 @@
+<template>
```

Comment:
Looking at this page in the ui and wondering if we need to change the language a bit? At the top it says extraction qc and then in the page it says Which QC Results would you like to upload? and then you have to select extraction qc.

Reply:
Yes happy to, do you have any opinions on what would be the best wording for consistency?